packages/common-all/src/util/treeUtil.ts

```diff
@@ -163,6 +163,15 @@ export class TreeUtils {
       });
     }
 
+    const getNoteTitleOrError = (note: NoteProps) => {
+      const { error } = NoteUtils.validate(note);
+      if (!error) {
+        return note.title.toLowerCase();
+      } else {
+        return `BAD NOTE TITLE: ${note.fname}`;
```

Comment:
Let's return something that will get sorted at the bottom, or undefined. This won't propagate to the user.